### PR TITLE
Add Code of Conduct to Venue page

### PR DESCRIPTION
**Pull request by mribeirodantas**
Signed-off-by: Marcel Ribeiro-Dantas <mribeirodantas@seqera.io>

### CODE REVIEW

File: src/pages/venue.jsx [hunk 1]
```diff
@@ -262,6 +262,61 @@ const VanuePage = () => {
...
+                      </span>
+                    </List.Item>
+                    <List.Item className="typo-body">
+                      <span className="mr-2">
+                        Respect the privacy of others
+                      </span>
+                    </List.Item>
+                    <List.Item className="typo-body">
+                      <span className="mr-2">
+                        Comply with the instructions of Docker staff and alert us if something is wrong
```
Comment (ewels) [inline src/pages/venue.jsx:305]: This one maybe needs to be double checked 🙄 😅  Could maybe replace it with something a bit different, or remove it.

**Review by ewels — CHANGES_REQUESTED** [review]

File: src/pages/venue.jsx (hunk 1 above)
Comment (mribeirodantas) [reply src/pages/venue.jsx:305]: Woooowww sorry for that 🙈

**Review by ewels — APPROVED** [review]
LGTM 👍🏻

File: src/pages/venue.jsx [hunk 2]
```diff
@@ -262,6 +262,61 @@ const VanuePage = () => {
...
+                      </span>
+                    </List.Item>
+                    <List.Item className="typo-body">
+                      <span className="mr-2">
+                        Collaborate to create a safe and supportive environment for all participants
+                      </span>
+                    </List.Item>
+                    <List.Item className="typo-body">
+                      <span className="mr-2">
+                        Refrain from harassing speech, content, and other harassing behavior
```
Comment (snafees) [inline src/pages/venue.jsx:290]: maybe say "unwelcome behavior"?

Comment (ewels) [reply src/pages/venue.jsx:290]: Updated in https://github.com/nextflow-io/summit-website/commit/1587f2081cb29b0790b2f670e752b675dca043c5 👍🏻

**Pull request merged**